src/main/java/com/training/library/AddBookServlet.java

```diff
@@ -1,20 +1,19 @@
-import Entities.Book;
-import Service.DatabaseService;
+package com.training.library;
+
+import com.training.library.entity.Book;
+import com.training.library.service.DatabaseService;
 
 import javax.servlet.ServletException;
-import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.sql.Connection;
 
-@WebServlet("/add")
-public class AddServlet extends HttpServlet {
 
+public class AddBookServlet extends HttpServlet {
     DatabaseService databaseService = new DatabaseService();
```

Comment:
Don't forget to make fields `private`, remember about encapsulation. This rule is applicable for all classes.

Reply:
Consider to make a book servlet which will handle all operations related to books for reducing amount of servlets.